apps/trading/components/rewards-container/reward-card.tsx

```diff
@@ -92,6 +95,11 @@ const RewardCard = ({
   gameId?: string | null;
 }) => {
   const t = useT();
+  const { rankTable } = dispatchStrategy;
+
+  const result = usePayoutPerRank(rankTable);
```

Comment:
Why not destructure immediately? Its a little cleaner imo

Reply:
alright